src/index.html

```diff
@@ -1,13 +1,658 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="ua" class="page">
```

Comment:
Change the lang attribute value to 'en' instead of 'ua', as 'en' is the standard language code for English.